service/ziti-tunnel/service/state.go

```diff
@@ -293,7 +296,15 @@ func (t *RuntimeState) LoadIdentity(id *Id, refreshInterval int) {
 		log.Infof("connecting identity completed: %s[%s] %t/%t", id.Name, id.FingerPrint, id.MfaEnabled, id.MfaNeeded)
 	}
 
-	id.CId = cziti.NewZid(sc)
+	rc := func(minTimeout int32, maxTimeOut int32) bool {
```

Comment:
why not jsut replace this whole thing with a function declared in sdk.go as: 
```
func (zid *ZIdentity) RefreshNeeded() bool {
	return zid.MinTimeout == -1 && zid.MaxTimeout == -1
}
``` ?  seems cleaner to me - is there a reason you had to add a function param?

Reply:
I will move it to sdk.go